i18n/en/docusaurus-plugin-content-docs/current/reference/units/layers/index.md

```diff
@@ -15,7 +15,7 @@ The first level of application partitioning, according to the **scope of influen
     ├── app/                    # Initializing application logic
     ├── processes/              # (Optional) Application processes running over pages
     ├── pages/                  # Application pages
-    ├── widgets/                # Independent and self-contained blocks for pages
+    ├── widgets/                # Self-sufficient UI blocks emerged from the composition of lower-level units in order to provide a complex user experience
```

Comment:
issue: I'm a bit concerned with "complex user experience" because it might scare people away from creating simpler widgets which are still an essential part of good decomposition

In your example below you show how to compose a post card, I wouldn't say that's very complex in terms of user experience. To me, "complex user experience" is something like the Monaco code editor or drag-n-drop lists or gesture recognition. 

suggestion: 
```suggestion
    ├── widgets/                # Self-sufficient UI blocks emerged from the composition of lower-level units
```

Reply:
@illright By saying "complex user experience", I mean that the resulting element covers several ( > 1 ) use cases, which at least involves combining different business logic units of type `Entity` and / or `Feature` in order to be implemented. I'm sure that it's really important to mention, so I'll try to express this definition differently 🤔

Reply:
I would argue that widgets don't necessarily have to serve >1 use case/business requirement. They are most useful in those cases, sure, but even with a single use case, they can serve as a nice compositional tool to organize code in small applications.

Basically, I want the definition to cover both simple apps and complex ones